### PR TITLE
FOLIO-1547 Use stripes 1.0

### PR DESCRIPTION
**Pull request by cherewaty**
Uses new `@folio/stripes` package.

### CODE REVIEW

**Comment by cherewaty** [comment]
Outstanding issues: `AutoSuggest` and `FieldRow` using some unexported `stripes-components` APIs.

**Comment by cherewaty** [comment]
Blocked by https://github.com/folio-org/stripes-smart-components/pull/291

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@folio/organization",
-  "version": "2.4.2",
+  "version": "2.4.0",
```
Comment (ttamj) [inline package.json]: Going down?

Comment (cherewaty) [reply package.json]: `2.4.0`, `2.4.1`, and `2.4.2` tags were never created in GitHub or on the `npm-folio` registry.

Comment (cherewaty) [reply package.json]: Bumped to `2.5.0` to prevent `npm-folioci` registry hiccups.

**Pull request merged**